nbs/15_migrate.ipynb

```diff
@@ -28,7 +28,8 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def _alias(fm:dict, p:Path):
</pre><p>I'd also prefer <code>fm</code> instead of <code>fmdict</code>  in the other notebooks too - more concise</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>